docs/tutorials/intro_tutorial.ipynb

```diff
@@ -4,23 +4,34 @@
+    "# Introductory Tutorial\n",
+    "\n",
+    "### The Boltzmann Wealth Model "
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "**Important:** \n",
+    "- If you are just exploring Mesa and want the fastest way to execute the code we recommend executing this tutorial online in a Colab notebook. [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/projectmesa/mesa/blob/main/docs/tutorials/intro_tutorial.ipynb)\n",
+    "- If you have installed mesa and are running locally, please ensure that your [Mesa version](https://pypi.org/project/Mesa/) is up-to-date in order to run this tutorial.\n",
+    "\n",
     "## Tutorial Description\n",
-    "Important: you must ensure that your Mesa version is up-to-date in order to run this tutorial.\n",
     "\n",
     "[Mesa](https://github.com/projectmesa/mesa) is a Python framework for [agent-based modeling](https://en.wikipedia.org/wiki/Agent-based_model). This tutorial will assist you in getting started. Working through the tutorial will help you discover the core features of Mesa.  Through the tutorial, you are walked through creating a starter-level model. Functionality is added progressively as the process unfolds. Should anyone find any errors, bugs, have a suggestion, or just are looking for clarification, [let us know](https://github.com/projectmesa/mesa/issues)!\n",
     "\n",
-    "The premise of this tutorial is to create a starter-level model representing agents exchanging money. This exchange of money affects wealth. Next, *space* is added to allow agents to move based on the change in wealth as time progresses.\n",
+    "The premise of this tutorial is to create a starter-level model representing agents exchanging money. This exchange of money affects wealth. \n",
+    "\n",
+    "Next, *space* is added to allow agents to move based on the change in wealth as time progresses.\n",
+    "\n",
+    "Two of Mesa's analytic tools: the *data collector* and *batch runner* are then used to examine the dynamics of this simple model. \n",
+    "\n",
+    "### More Tutorials: \n",
     "\n",
-    "Two of Mesa's analytic tools: the *data collector* and *batch runner* will be used to examine this movement. After that an *interactive visualization* is added which allows model viewing as it runs.\n",
+    "Visualization: There is a separate [visualization tutorial](https://mesa.readthedocs.io/en/stable/tutorials/visualization_tutorial.html) that will take users through building a visualization for this model (aka Boltzmann Wealth Model).\n",
     "\n",
-    "Finally, the creation of a custom visualization module in JavaScript is explored."
+    "Advanced Visualization: There is also an [advanced visualization tutorial](https://mesa.readthedocs.io/en/stable/tutorials/adv_tutorial_legacy.html) that will show users how to use the JavaScript based visualization option, which also uses this model as its base. "
```

Comment:
Needs to say this is legacy, because otherwise it seems to recommend that for advanced visualization, user should use the Tornado-based viz.

Reply:
Isn't this correct, for now? It's not like the experimental Viz is ready for advanced uses, just now. But I think in the advanced tutorial it should encourage users to use the new frontend, if they want to try it. 

But for now things like network graphs are easier to do/better documented in the legacy server

Reply:
> But I think in the advanced tutorial it should encourage users to use the new frontend, if they want to try it.

The naming for the Solara viz tutorial is called "visualization tutorial", and the old advanced tutorial is called "advanced visualization (legacy)". I decided to not call the Solara viz as "advanced visualization tutorial", because you don't need in-depth knowledge of Solara API to write what amounts to just a Matplotlib plot object.